docs/BUILD_SYSTEM.md

```diff
@@ -0,0 +1,78 @@
+# The Service-Catalog Build System
+
+This document aims to overview the build system in this repository.
```

Comment:
"Overview" is used like a verb here, which is awkward.

Consider "This document presents an overview of..."